ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js

```diff
@@ -33,22 +33,22 @@ describe("Form in site using embed container", () => {
         expect(formContainer.length).to.equal(2);
       });
 
-      it.skip("should submit on button click and thank you message of embed container must be overridden", () => {
+      it("should submit on button click and thank you message of embed container must be overridden", () => {
             cy.intercept({
             method: 'POST',
             url: '**/adobe/forms/af/submit/*',
         }).as('afSubmission');
-        cy.get(`.cmp-adaptiveform-button__widget`).eq(0).should('be.visible').click().then(() => {
+        cy.get(`.cmp-adaptiveform-button__widget`).eq(0).should('be.visible').wait(500).click().then(() => {
```

Comment:
Lets not add wait in the test case, it gets flaky with time, wait for an element instead

Reply:
Removed wait.